sdks/js/packages/core/react/components/organization/plans/hooks/usePlans.tsx

```diff
@@ -110,6 +115,10 @@ export const usePlans = () => {
     ]
   );
 
+  const checkBasePlan = (planId: string) => {
```

Comment:
Since this returns a boolean, can we rename it to `isBasePlan(planId)` for readability?

Reply:
I use the `is` prefix with boolean variables, not functions. for function, I use some verbs like `check` here